src/Generators/EntityGenerator.php

```diff
@@ -202,10 +204,11 @@ protected function getRelatedModels(string $model, string $creatableClass): arra
         $modelClass = $this->getModelClass($model);
 
         if (!class_exists($modelClass)) {
+            // TODO: pass $this->modelSubfolder to Exception after refactoring in https://github.com/RonasIT/laravel-entity-generator/issues/179
             $this->throwFailureException(
                 exceptionClass: ClassNotExistsException::class,
-                failureMessage: "Cannot create {$creatableClass} cause {$model} Model does not exists.",
-                recommendedMessage: "Create a {$model} Model by himself or run command 'php artisan make:entity {$model} --only-model'.",
+                failureMessage: "Cannot create {$creatableClass} cause {$this->model} Model does not exists.",
```

Comment:
it seems we need to use model here, as in the relation context it can be another model